control_plane/src/storage.rs

```diff
@@ -211,15 +192,35 @@ impl PageServerNode {
     }
 
     pub fn start(&self, config_overrides: &[&str]) -> anyhow::Result<()> {
-        print!(
+        self.start_node(config_overrides, &self.repo_path(), false)
+    }
+
+    fn start_node(
+        &self,
+        config_overrides: &[&str],
+        datadir: &Path,
+        update_config: bool,
+    ) -> anyhow::Result<()> {
+        println!(
```

Comment:
Note that the old code used `print!()` rather than `println!()`, so this will print an extra empty line

Reply:
Yes, yet I've found that more natural when dealing with failing pageserver: 

with empty line:
```
Work/neon/neon kb/pageserver-no-init:origin/kb/pageserver-no-init*​ ⇡❯ rm -rf .neon/ && ./target/debug/neon_local init
Starting pageserver at '127.0.0.1:64000' in '.neon'
thread 'main' panicked at '((((((((((((((((', pageserver/src/bin/pageserver.rs:41:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:142:14
   2: pageserver::main
             at ./pageserver/src/bin/pageserver.rs:41:9
   3: core::ops::function::FnOnce::call_once
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/ops/function.rs:248:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
pageserver init failed: Pageserver failed to start. See console output and '.neon/pageserver.log' for details.
```

without:
```
Work/neon/neon kb/pageserver-no-init:origin/kb/pageserver-no-init*​ ⇡❯ rm -rf .neon/ && ./target/debug/neon_local init
Starting pageserver at '127.0.0.1:64000' in '.neon'thread 'main' panicked at '((((((((((((((((', pageserver/src/bin/pageserver.rs:41:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:142:14
   2: pageserver::main
             at ./pageserver/src/bin/pageserver.rs:41:9
   3: core::ops::function::FnOnce::call_once
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/ops/function.rs:248:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
pageserver init failed: Pageserver failed to start. See console output and '.neon/pageserver.log' for details.
```